### ISSUE TITLE
Install pyqg kernel issue 

### ISSUE THREAD

**Issue by WillyChap** [pyqg/pyqg#220]
I have set up a python2 environment and following your instructions from your install page. As detailed below:

 `conda create --yes -n test_env python=2.7 pip nose numpy cython scipy nose`

The first problem is here. it seems that the nanshe channel is no longer available, so I've switched to conda-forge. That seems to do the trick. 
`conda install --yes -n test_env -c nanshe pyfftw`
error:
```
PackagesNotFoundError: The following packages are not available from current channels:

  - pyfftw

Current channels:

  - https://conda.anaconda.org/nanshe/osx-64
  - https://conda.anaconda.org/nanshe/noarch
  - https://repo.anaconda.com/pkgs/main/osx-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

```

instead: 
`conda install --yes -n test_env -c conda-forge pyfftw`
this seems to install well: 
```
$python -c 'import pyfftw as pft; print(pft.__version__);'
$0.12.0
```
However when I run: 

```
$ conda activate test_env
$pip install pyqg

```
I get this error with visioneer: 
```
Using cached https://files.pythonhosted.org/packages/be/22/5063041125d40e3807a272c45227cdec43910a6f75d8bee28a7f2eed7570/pyqg-0.3.0.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /Users/will/anaconda2/envs/test_env/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/bk/vd0dgqrn3wz8pfjphf_ztvl40000gn/T/pip-install-bTz9Pa/pyqg/setup.py'"'"'; __file__='"'"'/private/var/folders/bk/vd0dgqrn3wz8pfjphf_ztvl40000gn/T/pip-install-bTz9Pa/pyqg/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/bk/vd0dgqrn3wz8pfjphf_ztvl40000gn/T/pip-install-bTz9Pa/pyqg/pip-egg-info
         cwd: /private/var/folders/bk/vd0dgqrn3wz8pfjphf_ztvl40000gn/T/pip-install-bTz9Pa/pyqg/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/bk/vd0dgqrn3wz8pfjphf_ztvl40000gn/T/pip-install-bTz9Pa/pyqg/setup.py", line 7, in <module>
        import versioneer
    ImportError: No module named versioneer
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```
Instead I cloned the github repo:
```
$ git clone https://github.com/pyqg/pyqg.git
$ python setup.py install
```
I get one warning regarding the kernel, but it installs: 
```
/Users/will/anaconda2/envs/test_env/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: "Using deprecated NumPy API, disable it with "
      "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
#warning "Using deprecated NumPy API, disable it with " \
 ^
```

However importing pyqg in python:

```
 
$ python
>>> import pyqg 
>>> import pyqg
> 




Traceback (most recent call last):

  File "<stdin>", line 1, in <module>

  File "pyqg/__init__.py", line 2, in <module>

    from .model import Model

  File "pyqg/model.py", line 8, in <module>

    from .kernel import PseudoSpectralKernel, tendency_forward_euler, tendency_ab2, tendency_ab3

ImportError: No module named kernel

```

Thank you for any help you can offer!

**Comment by WillyChap** [comment]
This is how to resolve this issue. 

The pip install was kicking the above issue, so we took to git, but continued to run into library linking issues during compile. 

**To solve:** 
After cloning the library:
` git clone https://github.com/pyqg/pyqg.git`

navigate to ./pyqg/model.py 

and add before you import  .kernel: 
```

import pyximport
pyximport.install()


from .kernel import PseudoSpectralKernel, tendency_forward_euler, tendency_ab2, tendency_ab3

```

then in setup.py we needed to extra link to a library after the defined readme() function: 

```
def readme():
    with open('README.md') as f:
        return f.read()

extra_link_args.append("-liomp5_db")

```

this seemed to fix my issue which was actually linked back to a shared object library. A list of dependencies would be great!
